routes/web.php

```diff
@@ -29,3 +29,9 @@
    'index' => 'courses',
 ]);
 
+Route::get('/users', 'UserController@index');
+
+Route::get('/edit/user/', 'UserController@edit')->name('user.edit');
```

Comment:
This is where you should implement auth middleware.

Reply:
https://laravel.com/docs/7.x/middleware